fdbserver/GetEncryptCipherKeys.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * GetCipherKeys.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBSERVER_GETCIPHERKEYS_H
+#define FDBSERVER_GETCIPHERKEYS_H
+#pragma once
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include "fdbserver/ServerDBInfo.h"
+#include "flow/BlobCipher.h"
+
+extern Future<std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>>> getLatestEncryptCipherKeys(
+    const Reference<AsyncVar<ServerDBInfo> const>& db,
+    const std::unordered_map<EncryptCipherDomainId, EncryptCipherDomainName>& domains);
+
+extern Future<std::unordered_map<BlobCipherDetails, Reference<BlobCipherKey>>> getEncryptCipherKeys(
```

Comment:
a nit: seems like this method is overloaded, either we add API documentation to make its usage clear OR best is to avoid function overloading and use explicit function names.

Reply:
Updated API documentation. Look better?